### DISCUSSION TITLE
Errors should not be logged at the debug level

### DISCUSSION THREAD

**Discussion by AaldertVanWijk-TomTom** [micronaut-projects/micronaut-core#7355] [Ideas]
We had a serialization issue in the code that caused Micronaut to return a partially serialized object with a status of 200 (something I consider wrong in and of itself)

Part of what made this hard to find and debug (and prevents our monitoring reacting on error logs) is that this pretty severe error is deemed only worth printing when the log level is set to `debug`
This seems like a bad choice to me. and something that should be rectified. 

https://github.com/micronaut-projects/micronaut-servlet/blob/5a32cab0f06a634ad3ffc916c2d0c9862022c880/servlet-core/src/main/java/io/micronaut/servlet/http/ServletHttpHandler.java#L521

(I see this happens on 4 different occasions in the same class. so I thought it must be some sort design choice. Hence the opening of a discussion)

**Comment by graemerocher** [comment]
please create an issue

**Reply by AaldertVanWijk-TomTom** [reply]
https://github.com/micronaut-projects/micronaut-core/issues/7358
